packages/app-desktop/gui/NoteEditor/utils/contextMenu.ts

```diff
@@ -106,8 +106,12 @@ export function menuItems(dispatch: Function): ContextMenuItems {
 				if (!options.filename) {
 					throw new Error('Filename is needed to save as png');
 				}
+				const [width,height] = svgDimensions(options.textToCopy);
+				if (!Number.isInteger(width) || !Number.isInteger(height)) {
+					bridge().showErrorMessageBox(_('failed to get width and height of image'));
```

Comment:
I think, with this change, `<svg xmlns="http://www.w3.org/2000/svg"><rect x='5' y='5' width='50' height='50' fill='red' stroke='black' style='stroke-width: 5;'/></svg>` would show an error message when it wouldn't before, because of the missing `viewBox` attribute.